### ISSUE TITLE
community logins bundle alert

### ISSUE THREAD

**Issue by gitforce** [dhoechst/salesforce-limit-monitor#8]
Dear Daniel,
Can the app be configure to alert for reaching close (80%) of the API calls consumed for a community logins bundle (2000 API calls) so a new bundle can be purchased before hand.
With much Appreciation,
gitForce

**Comment by dhoechst** [comment]
Do you mean number of community logins or API calls associated with those logins? I'm not aware of bundle of API calls that come with community logins. I have a separate issue #4 opened to monitor usage based entitlements which is where the login information is tracked.

**Comment by gitforce** [comment]
Dear Daniel,
To give a general idea.
We use community as the basis for recruiting  candidates to our university
academic staff.
Each candidate needs a license for a very limited number of logins.
That is why we made an advantage of the 2000 logins/per year bundle option.
The limit and actual consumption is shown in company information under
licenses section.
We happened to hit the limit one month earlier :)) and our candidates have
been blocked for several days before we even knew that was the reason.
Then I remembered the wonderful podcast you’ve taken part and wonder if
salesforce limits can handle in its current state something as such.

Kindest regards,
Itay
It would be
בתאריך יום ב׳, 6 ביולי 2020 ב-19:28 מאת Daniel Hoechst <
notifications@github.com>:
